### DISCUSSION TITLE
Using open telemetry logback  appender for our application code

### DISCUSSION THREAD

**Discussion by debashish-github** [open-telemetry/community#1367] [Community Discussions]
Hi..
   We are currently in the process of transitioning to open telemetry for our application logging .
Our application is a spring boot java web application that currently uses Logback for logging.
We want to seamlessly move our logging to open telemetry logback appender so that the logs are send to the OLTPExporter .
My understanding is that all our logging in different classes of our application using logback logger should not change. 
Only the globalAppender needs to be setup correctly to make this happen which is a very minimal code change .
I was going through the documentation provided at https://github.com/open-telemetry/opentelemetry-java-instrumentation/tree/main/instrumentation/logback/logback-appender-1.0/library .
I have done the configuration setup on my app . The last step asks to add the following steps 
SdkLoggerProvider sdkLoggerProvider =
  SdkLoggerProvider.builder()
    .setResource(Resource.create(...))
    .addLogProcessor(...)
    .build();
GlobalLoggerProvider.set(sdkLoggerProvider);

I currently have the following dependency in my pom
		  <dependency>
		    <groupId>io.opentelemetry.instrumentation</groupId>
		    <artifactId>opentelemetry-logback-appender-1.0</artifactId>
		    <version>1.22.1-alpha</version>
		    <scope>runtime</scope>
		  </dependency>

Which other dependency is required for me to implement this ? Also which dependency is required for OLTP Exporter ?
Any help is greatly appreciated !

Thanks

**Comment by trask** [comment]
hi @debashish-github! I'd recommend opening this in the [java-instrumentation repository](https://github.com/open-telemetry/opentelemetry-java-instrumentation) repository
